scheduler/scheduler.go

```diff
@@ -250,6 +276,7 @@ func BuildDependencyGraph(c client.Interface, sel labels.Selector) (DependencyGr
 			depGraph[child].Requires, depGraph[parent])
 
 		depGraph[child].Meta[parent] = d.Meta
+		depGraph[parent].Meta[parent] = d.Meta
```

Comment:
But with multiple children same You will in result have only last value for this parent. Am I right?
IMO Ilya should also take a look on this.

Btw. I hate inline comments (they are quite good markers that code needs to be refactored into simpler one, but better understandable) - but this is clearly a good place for such comment.